### PR TITLE
add freeAll to proxy reducer, and call it when user clicks set max

### PR DESCRIPTION
**Pull request by b-pmcg**

### CODE REVIEW

File: src/components/modals/shared/AmountInput.js [hunk 1]
```diff
@@ -24,11 +25,12 @@ export default class AmountInput extends Component {
   }
 
   setAmount = event => {
+    // TODO: use freeAll if user inputs the max amount?
```
Comment (b-pmcg) [inline src/components/modals/shared/AmountInput.js]: @jparklev do we want to call free all when the user manually enters the max amount?

File: src/reducers/proxy.js [hunk 2]
```diff
@@ -69,15 +73,18 @@ const handleTx = ({
...
     txMgr.listen(txObject, {
       pending: tx => {
+        console.log('handleTx pending');
         dispatch({
           type: `proxy/${prefix}_SENT`,
           payload: { txHash: tx.hash }
         });
       },
       mined: async _ => {
+        console.log('handleTx mined');
```
Comment (b-pmcg) [inline src/reducers/proxy.js]: I need to look more into why the loading spinner isn't popping up when the tx begins, and doesn't resolve when its been mined.

File: src/components/modals/shared/AmountInput.js (hunk 1 above)
Comment (jparklev) [reply src/components/modals/shared/AmountInput.js]: good question. My feeling is that we should. What do you think?

Comment (b-pmcg) [reply src/components/modals/shared/AmountInput.js]: Seems like good practice, and I can't think of any particular reason not to do so.

File: src/components/VoterStatus.js [hunk 3]
```diff
@@ -111,6 +111,7 @@ const VoterStatus = ({
   signaling,
   onboardingState
 }) => {
+  console.log('voter status component, fetching?', fetching);
```
Comment (b-pmcg) [inline src/components/VoterStatus.js]: @jparklev I'm kind of stumped on this. When calling `free()`, after the tx is mined, this VoterStatus component re-renders to display the updated balances. When calling `freeAll()`, after the tx is mined, VoterStatus does not get re-rendered, and the new balances are not displayed. I think there is a dispatch tied to it somewhere, but I can't seem to find the thread. I'll keep looking into it, but wondered if you had any thoughts?

Comment (jparklev) [reply src/components/VoterStatus.js]: ah, i believe those changes are happening here
https://github.com/makerdao/governance-dashboard/blob/d1d365d5c3fbb0f9d1e0f11e0b26616712ceb9e4/src/reducers/accounts.js#L538
the `accounts` reducer is listening for the `WITHDRAW_MKR_SUCCESS` action and updating it's own store in response

Comment (b-pmcg) [reply src/components/VoterStatus.js]: ah, thanks! That was it, keen eye.

**Pull request merged**